src/app/services/sms/sms.errors.ts

```diff
@@ -1,9 +1,16 @@
 import { ApplicationError } from '../../modules/core/core.errors'
 
 export class SmsSendError extends ApplicationError {
+  code?: number
+  sendStatus: unknown
+
   constructor(
     message = 'Error sending OTP. Please try again later and if the problem persists, contact us.',
+    code?: number,
+    sendStatus?: unknown,
```

Comment:
To clarify, why is type of sendStatus unknown, shouldn't it be string?
```
type MessageStatus = 'queued'|'sending'|'sent'|'failed'|'delivered'|'undelivered'|'receiving'|'received'|'accepted'|'scheduled'|'read'|'partially_delivered';
```

Reply:
No need to add a type since we are not going to do anything with the status anyways; it is just for logging. I also do not want to constraint `SmsError` to Twilio types